bin/test-integration

```diff
@@ -82,20 +90,21 @@ def run_scenario(scenario)
     Bundler.with_original_env do
       run_cmd(scenario, "bundle check || bundle install")
 
-      # set up the database and schema files
-      run_cmd(scenario, "bin/rails db:prepare")
+     adapter_config = {"RAILS_ENV" => "test","ARTENANT_SCHEMA_DUMP" => "1"}
```

Comment:
On `main`, the integration test suite runs `rails db:prepare` in the development environment, and then the tests get run in the test environment (implicitly setting up the databases).

Can you say a bit more about this change?